src/webgpu/shader/execution/builtin/min.spec.ts

```diff
@@ -21,6 +21,21 @@ import { anyOf, Case, Config, correctlyRoundedThreshold, kBit, run } from './bui
 
 export const g = makeTestGroup(GPUTest);
 
+/** Generate set of min test cases from an ascending list of values */
+function generateTestCases(test_values: Array<Scalar>) {
```

Comment:
Might be nice to include a return type on this function signature.

Reply:
Done